src/pydata_sphinx_theme/assets/styles/sections/_header.scss

```diff
@@ -107,10 +107,26 @@
         border: 1px solid var(--pst-color-border);
         box-shadow: 0 0 0.3rem 0.1rem var(--pst-color-shadow);
         background-color: var(--pst-color-on-background);
-        padding: 0.5rem 1rem;
+        padding: 0.5rem 0;
         margin: 0.5rem 0;
         min-width: 20rem;
 
+        .dropdown-item {
+          // Give the items in the dropdown some breathing room but let the hit
+          // and hover area of the items extend to the edges of the menu
+          padding: 0.25rem 1.5rem;
+
+          // Override Bootstrap
+          &:focus:not(:hover):not(:active) {
+            background-color: inherit;
+          }
+        }
+
+        // Override Bootstrap
+        .nav-link {
+          transition: none;
```

Comment:
This one was making me go a little batty. Because at first I couldn't tell what was still different between this dropdown and the other, but then when I looked closer, I realized that the items in the version switcher dropdown do not have the `nav-link` class, which comes with a {color, background-color, border-color} transition from Bootstrap. But this meant that the {text-decoration: underline} of the links would occur ever so slightly faster than the color changes.

I think this was creating some weird effect, where it seemed like the text color change was lagging the roll-over/hover background color change or vice versa.

Turning off the transitions makes this dropdown match the other one. However, it means that the `nav-link` links in this dropdown do not transition their color like the ones before the dropdown in the nav bar. 

In other words, with this change, if we have the following links A, B, C, with A and B in the nav bar outside the dropdown, and link C is in the dropdown, then both links A and B will have a subtle text color transition when you roll the mouse over them, whereas link C will not.

Reply:
for consistency why not turn off the transition for all `nav-link`s (not just the ones in the dropdown)?

Reply:
tl;dr-- My opinion: we should either disable the transition, as you suggest @drammock, or find a way to get them consistent and in sync.

The reason I didn't set `transition: none` for all `.nav-link` elements is that I assume that most of the Bootstrap styles are generally desired, since Bootstrap is a mature library, and extensive thought has been put into the design details.

That said, your question made me realize something. We have added [hover-underlining to the `nav-link` class](https://github.com/trallard/pydata-sphinx-theme/blob/40c79bbdc2b8085b542ef0c6cd69420db2df0c8e/src/pydata_sphinx_theme/assets/styles/sections/_header.scss#L131). That's not something Bootstrap provides. In fact, we actually override Bootstrap's default behavior, which is to [explicitly **not** underline](https://github.com/twbs/bootstrap/blob/961d5ff9844372a4e294980c667bbe7e0651cdeb/scss/_nav.scss#L36) nav links on hover.

_hours later ..._

😩

Turns out, getting the text underline to fade in and out, so that it looks in sync with the color transitions that Bootstrap puts on `.nav-link` elements, is really, really hard[^*]. But the reason it matters is that if the underline appears and disappears before the color transition finishes, even with only a 0.15s lag, I think it makes the overall effect just a tiny bit funky and off-putting, at least to my brain. 

So... my recommendation: either we turn off the transition altogether, or we find a way to get the underline to animate (somehow) in sync with the 0.15s color transition. 

cc @trallard because the git history shows that you have thought a lot about the link hover styles.

This is ultimately a microscopic design detail, but since it's come up in this PR, I feel like we might as well address it and come to a decision about what path we want to take. I created a Codepen to put together what I see as all the [possible paths forward](https://codepen.io/gabalafou-the-looper/pen/mdadzxm?editors=1100).

[^*]: It's hard because text-decoration-style is a discrete animation property, meaning it goes on and off like a lightbulb; it's not natively transitionable. I tried using text-decoration-thickness, but it seems that you can't actually set a thickness of 0px? Or at least, when I did set thickness to 0, I got a really thin but not zero-width underline. I'm also not really sure if animating the line thickness—having the underline grow from thinner to thicker—is really a good way to make the color transition sync with the underline transition. (Also the transition just straight up doesn't seem to work in Chrome, even though it should? It does work in Firefox. 🤷‍♂️) I also tried using a `:before` pseudo-element, but that had all sorts of alignment issues that I wasn't willing to throw hours of time into trying to solve. I also tried animating a bottom border from transparent to a solid color, but that's also tricky to guarantee that the border only extends the length of the text (this is where text-decoration really outshines border-bottom). Maybe some CSS wizard knows the right incantation. I do not.

Reply:
Had a chat with @trallard and she's in favor of turning off all the .nav-link transitions for now. 

That makes three of us, so I'm going to go ahead and push that change to this PR.

We can revisit this in the future once higher priority design and accessibility issues have been resolved.